test/unit/test_attention.py

```diff
@@ -119,7 +119,7 @@ def test_coverage_attention(attention_coverage_type,
     attention_prob_result = exec_output[1].asnumpy()
     dynamic_source_result = exec_output[2].asnumpy()
 
-    expected_probs = (1 / source_length_vector).reshape((batch_size, 1))
+    expected_probs = (1. / source_length_vector).reshape((batch_size, 1))
```

Comment:
I think this kind of thing is deprecated in Python 3. It's no longer necessary since floating point division is the default, and you have to force integer division with `//`.

Reply:
Sure but it also doesn't hurt right? Anyway, source_length_vector is a numpy array so I am not sure the change in Python3 would also apply to numpy.